documentation/functions.asciidoc

```diff
@@ -298,13 +298,16 @@ The first three parameters are taken directly into variables and removed from th
 If the last parameter is a single directory and, on MacOS, not a directory with a name like *.app, it is taken as the source directory.
 Then the function doReplaceExtractedFile is called with the saved parameters, with source directory, target directory and backup directory.
 
-=== doUpdateMirrors
-Uses xref:doGitPullOrClone[] to clone or pull `mirrors` ensuring that everything is up-to-date.
-
 === doUnzip
 Checks if an unzip program is installed, and if not, installs the program.
 The file specified in the first parameter is then unpacked into the directory specified in the second parameter.
 
+=== doUpdateMirrors
+Uses xref:doGitPullOrClone[] to clone or pull `mirrors` ensuring that everything is up-to-date.
+
+=== doUpdateUrls
+Uses xref:doGitPullOrClone[] to clone or pull `urls` ensuring that everything is up-to-date.
```

Comment:
Great that you reworked the functions avoiding side effects via global variables.
Can you also update the documentation now?